<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QPANSOPYLNAVDockWidgetBase</class>
 <widget class="QDockWidget" name="QPANSOPYLNAVDockWidgetBase">
  <property name="windowTitle">
   <string>QPANSOPY LNAV</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="approachTypeGroup">
      <property name="title">
       <string>Approach Type</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="initialRadioButton">
          <property name="text">
           <string>Initial Approach</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="intermediateRadioButton">
          <property name="text">
           <string>Intermediate Approach</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="finalRadioButton">
          <property name="text">
           <string>Final Approach</string>
          </property>
         </widget>
        </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="layersGroupBox">
      <property name="title">
       <string>Input Layer</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
          <widget class="QLabel" name="routingLayerLabel">
           <property name="text">
            <string>Routing Layer:</string>
           </property>
          </widget>
        </item>
        <item row="0" column="1">
          <widget class="QgsMapLayerComboBox" name="routingLayerComboBox"/>
        </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="outputGroupBox">
      <property name="title">
       <string>Output</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="outputFolderLabel">
           <property name="text">
            <string>Output Folder:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="outputFolderLineEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="browseButton">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="exportKmlCheckBox">
         <property name="text">
          <string>Export to KML</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="calculateButton">
      <property name="text">
       <string>Calculate</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="logGroupBox">
      <property name="title">
       <string>Log</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTextEdit" name="logTextEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
</ui>
